src/sql/workbench/services/tableDesigner/browser/tableDesignerComponentInput.ts

```diff
@@ -31,9 +33,15 @@ export class TableDesignerComponentInput implements DesignerComponentInput {
 	public readonly onEditProcessed: Event<DesignerEditProcessedEventArgs> = this._onEditProcessed.event;
 	public readonly onStateChange: Event<DesignerStateChangedEventArgs> = this._onStateChange.event;
 
+	private readonly designerEditTypeDisplayValue: { [key: number]: string } = {
+		0: 'Add', 1: 'Remove', 2: 'Update'
```

Comment:
instead of 0,1,2 can we use the enum that you previously modified here?

Reply:
I need to use 0,1 , 2 because the enum targets 0, 1 , 2 for DesignerEditType

Reply:
that's what i meant, how about use DesignerEditType.Add instead of 0 here

Reply:
You cant put enum fields like the left side like that